dm/checker/binlog.go

```diff
@@ -0,0 +1,174 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package checker
```

Comment:
maybe move this package to  `ticdc/pkg/checker`  or `ticdc/dm/pkg/` ? https://github.com/golang-standards/project-layout#pkg

Reply:
done in https://github.com/pingcap/ticdc/pull/3770/commits/7b672e89a4116ee2adfbe68645084a69ae091c5c